lib/main.ts

```diff
@@ -48,6 +47,7 @@ export const COMPILER_OPTIONS: ts.CompilerOptions = {
   experimentalDecorators: true,
   module: ts.ModuleKind.CommonJS,
   target: ts.ScriptTarget.ES5,
+  moduleResolution: ts.ModuleResolutionKind.Classic,
```

Comment:
do you need this for ts2dart? I don't think that you have any named imports pointing to your source directory